Weapon-Master/Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    public float moveSpeed;
+
+    Rigidbody rb;
+    CapsuleCollider capsuleCollider;
+
+    void Awake() {
+        rb = GetComponent<Rigidbody>();
+        capsuleCollider = GetComponent<CapsuleCollider>();
+    }
+    
+    void Start()
+    {
+        
+    }
+
+    void Update()
+    {
+        
+    }
+
+    void FixedUpdate() {
+        Move();
+    }
+
+    void Move()
+    {
+        float moveDirX = Input.GetAxisRaw("Horizontal");
+        float moveDirZ = Input.GetAxisRaw("Vertical");
+
+        Vector3 moveHorizontal = transform.right * moveDirX;
+        Vector3 moveVertical = transform.forward * moveDirZ;
+
+        Vector3 velocity = (moveHorizontal + moveVertical).normalized * moveSpeed;
+
+        rb.MovePosition(transform.position + velocity * Time.deltaTime);
```

Comment:
-**Move()** might some seperations since it has both input and rigidbody method.
-Since **.nornalized** returns only **0 and 1**, player won't stop immediately. Not sure but maybe **Clamp** would work.